plexapi/utils.py

```diff
@@ -176,7 +176,11 @@ def toDatetime(value, format=None):
     """
     if value and value is not None:
         if format:
-            value = datetime.strptime(value, format)
+            try:
+                value = datetime.strptime(value, format)
+            except ValueError:
+                # parsing failed
```

Comment:
LOG.info('Fail to parse %s to datetime, defaulting to None', value)

Reply:
Yes, of course.
I'll submit a new PR.